lib/Sema/TypeCheckDecl.cpp

```diff
@@ -2930,7 +2930,7 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
 
     if (!IsSecondPass) {
       for (unsigned i = 0, e = PBD->getNumPatternEntries(); i != e; ++i) {
-        // Type check each VarDecl in that his PatternBinding handles.
+        // Type check each VarDecl in that their PatternBinding handles.
```

Comment:
I think this is supposed to say "Type check each VarDecl **that this** PatternBinding handles.".